service/src/main/resources/db/postgres/V3_0_0__add_attribute_connectors.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE attribute_connector (
+  id bigserial NOT NULL primary key,
+  refresh_interval_hours integer DEFAULT 0
+);
+
+CREATE TABLE attribute_adapter_connection (
+  id bigserial NOT NULL primary key,
+  connector_id integer NOT NULL,
+  adapter_endpoint varchar(128) NOT NULL,
+  adapter_token_url varchar(128) NOT NULL,
+  adapter_client_id varchar(128) NOT NULL
+);
+
+ALTER TABLE authorization_zone ADD COLUMN resource_attribute_connector integer NULL;
```

Comment:
Agreed that foreign keys must have the same type of the referenced column's type.  However, for postgres attribute_connector(id) is defined as bigserial.  We can't use this type when defining a nullable foreign key for zone entity as bigserial does not support nullability.  Since bigserial is just a wrapper for bigint to support auto-generating sequencing, decided to use bigint for foreign key definition.